index.html

```diff
@@ -2716,14 +2716,14 @@ <h2 id="OfflineAudioCompletionEvent">
             <pre class="example">
     nodeA.connect(nodeB);
     nodeA.connect(nodeB);
-    
+
```

Comment:
Why the extra blank line?  Did tidy do this?

Same comment applies to all the other extra blank lines below at the end of a `<pre>` section.